defaults/main.yml

```diff
@@ -527,6 +527,12 @@ gitlab_runner__cache_secret_key: ''
 gitlab_runner__cache_bucket_name: ''
 
                                                                    # ]]]
+# .. envvar:: gitlab_runner__cache_bucket_location [[[
+#
+# Cache bucket location
+gitlab_runner__cache_bucket_location: 'us-east-1'
```

Comment:
I suppose that if it's default, it can stay. The variable is not active without the rest of them, I suppose. Are you sure that it won't interfere when Runner is not configured with S3 support?